src/cheri_riscv_types.sail

```diff
@@ -1,28 +1,53 @@
-/* Extension for CHERI PTE checks */
-enum pte_check = {PTE_CAP_OK, PTE_LOAD_CAP_ERR, PTE_STORE_CAP_ERR}
+/*
+ * Extension for CHERI page table walker
+ *
+ * We are going to carry around a summary of the PTW walk for data-dependent
+ * (well, tag-dependent) behaviors by the instructions.
+ *
+ */
 
-/* Extension for CHERI page-table-walks */
-
-enum ext_ptw = {PTW_CAP_OK, PTW_LOAD_CAP_ERR, PTW_STORE_CAP_ERR}
+enum ext_ptw_lcm = {
+  PTW_LCM_OK /* Tags flow as might be expected */,
```

Comment:
I'd put commas before the comment, on first read it looked like they were missing.